frontend/src/components/NavList/NavList.jsx

```diff
@@ -1,23 +1,22 @@
-import { Router } from 'next/router';
 import React from 'react';
-import { useRouter } from 'next/router'
+import auth from '../../hooks/auth';
 
 import styles from './NavList.module.css';
 
 function NavList({loggedIn}) {
-    const router = useRouter()
-    const handleLoguout = ()=>{
-        localStorage.clear()
-        router.push('/sign-in')
+
+    const handleLogout = ()=>{
+        auth.logout()
     }
```

Comment:
> Should the `logout()` function be classified as a hook if it's [not called at the top level](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level)?

I don't see why it should be a hook. Because it's just clearing localstorage for now.

Reply:
I'm only raising this up because it's placed inside of a hooks folder.